powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationRecordEntity.java

```diff
@@ -41,635 +43,206 @@
  */
 @Entity
 @Table(name = "pa_activation")
+@Getter @Setter
 public class ActivationRecordEntity implements Serializable {
 
     @Serial
     private static final long serialVersionUID = 7512286634644851705L;
 
+    /**
+     * Activation ID.
+     */
     @Id
     @Column(name = "activation_id", length = 37)
     private String activationId;
 
+    /**
+     * Activation code.
+     */
     @Column(name = "activation_code", nullable = false, updatable = false)
     private String activationCode;
 
+    /**
+     * Activation OTP validation.
+     */
     @Column(name = "activation_otp_validation", nullable = false)
     @Convert(converter = ActivationOtpValidationConverter.class)
     private ActivationOtpValidation activationOtpValidation;
 
+    /**
+     * Activation OTP.
+     */
     @Column(name = "activation_otp")
     private String activationOtp;
 
+    /**
+     * External ID.
+     */
     @Column(name = "external_id")
     private String externalId;
 
+    /**
+     * User ID.
+     */
     @Column(name = "user_id", nullable = false, updatable = false)
     private String userId;
 
+    /**
+     * Activation name.
+     */
     @Column(name = "activation_name")
     private String activationName;
 
+    /**
+     * Extra parameter.
+     */
     @Column(name = "extras", columnDefinition = "CLOB")
     private String extras;
 
+    /**
+     * Protocol.
+     */
     @Convert(converter = ActivationProtocolConverter.class)
     @Column(name = "protocol", nullable = false, columnDefinition = "varchar(32) default 'powerauth'")
     private ActivationProtocol protocol;
 
+    /**
+     * User device platform.
+     */
     @Column(name = "platform")
     private String platform;
 
+    /**
+     * User device information.
+     */
     @Column(name = "device_info")
     private String deviceInfo;
 
+    /**
+     * Activation flags.
+     */
     @Column(name = "flags")
     @Convert(converter = ActivationFlagConverter.class)
     private final List<String> flags = new ArrayList<>();
 
+    /**
+     * Base64 encoded server private key
```

Comment:
Inconsistent usage of periods at the end of sentence in this class.

Reply:
```suggestion
     * Base64 encoded server private key.
```